indico/modules/events/abstracts/templates/emails/abstract_accept_notification.txt

```diff
@@ -0,0 +1,19 @@
+{% extends 'emails/base.txt' %}
+
+{% block subject -%}
+    Abstract Acceptance notification (#{abstract_id})
+{%- endblock %}
+
+{% block header_recipient -%}
+    {submitter_name}
+{%- endblock %}
+
+{% block body -%}
+    {%- filter dedent -%}
+        We're pleased to announce that your abstract "{abstract_title}" with ID #{abstract_id} has been accepted
+        as a {contribution_type} in track "{abstract_track}".
```

Comment:
"as a {contribution_type}" only sounds ok if it's e.g. "poster" - but with something like "oral" it sounds weird. maybe `"accepted in the track "<track>" (<type>)` or something like that?